docs/examples/apt-cacher-ng.md

```diff
@@ -53,7 +55,7 @@ use:
     $ docker logs -f test_apt_cacher_ng
 
 To get your Debian-based containers to use the proxy, you can do one of
```

Comment:
`do one of the five things` feels a liitle bit weird ? -> `You have the following options` (I think we don't even need to count them :stuck_out_tongue_winking_eye:).